### PR TITLE
Updated Readme

### PR DESCRIPTION
**Pull request by Jassi10000**
Mentioned more clearly about what type of database we will actually be using in  mentioned MERN Stack

### CODE REVIEW

File: ReadMe.md
Comment (praveenscience) [inline ReadMe.md:39]: Make it a single line, please. Give a line space. Thanks for this! 😅 Also, why `[ ]` for "We have ..." Use a normal `()` without any spaces inside.

**Review by praveenscience — CHANGES_REQUESTED** [review]

**Comment by Jassi10000** [comment]
Done the changes told sir @praveenscience

File: ReadMe.md
Comment (praveenscience) [inline ReadMe.md:35]: Oh, I meant, in the source code view, do the following:

1. Add a line space after the heading.
2. Remove character spaces before each lines.
3. Keep everything what you have written originally in a single line, not multiline.

**Review by praveenscience — CHANGES_REQUESTED** [review]

File: ReadMe.md
Comment (praveenscience) [inline ReadMe.md:36]: 1. Remove the HTML tags please.
2. Remove the space before the comma.
3. Add the complete text that I sent you.

**Review by praveenscience — CHANGES_REQUESTED** [review]

File: ReadMe.md
Comment (praveenscience) [reply ReadMe.md]: 1. Done
2. Not Done.
3. Not Done.

**Comment by Jassi10000** [comment]
image



Sir pls see my latest commit , I have done all the changes you told @praveenscience 
I have removed the space before comma  and also used the same text you sent to me , and this commit was made by me around an  hour ago , please check this ss for reference

**Comment by praveenscience** [comment]
@Jassi10000 I'll say you what exactly is done. 😉

**Comment by praveenscience** [comment]
@Jassi10000 Check the [diff now](https://github.com/praveenscience/Internship-LMS-FrontEnd/pull/22/commits/abb921cdc86daa33e4b9992bc938cb7f507ef44b). If you're using `[]` it becomes a syntax in Markdown. It's crucial to know about these.

**Review by praveenscience — APPROVED** [review]
LGTM! 👍

**Comment by Jassi10000** [comment]
> @Jassi10000 Check the [diff now](https://github.com/praveenscience/Internship-LMS-FrontEnd/pull/22/commits/abb921cdc86daa33e4b9992bc938cb7f507ef44b). If you're using `[]` it becomes a syntax in Markdown. It's crucial to know about these.

Yeah sure , thank you for that awesome knowledge 😀

**Comment by praveenscience** [comment]
@Jassi10000 We'll wait for the other maintainers to give their approval! 😁

**Review by Salonii02 — APPROVED** [review]

**Review by manudd1011 — COMMENTED** [review]
.

**Pull request merged**
